Documentation/devel/tpm.md

```diff
@@ -0,0 +1,15 @@
+# rkt and the Trusted Platform Module
+
+rkt supports *measuring* container state and configuration into the Trusted Platform Module (TPM) event log. Enable this functionality by building rkt with the [`--enable-tpm=yes` option to `./configure`][build-configure-tpm]. rkt accesses the TPM via the [`tpmd` executable available from the go-tspi project][go-tspi]. This `tpmd` is expected to listen on port 12041.
```

Comment:
Although `--enable-tpm` is `yes` by default, I guess it's fine to be explicit here.